### PR TITLE
[Improvement]: batch export confirmation

### PR DESCRIPTION
**Pull request by SamyMP**
## Changes in this pull request  
Similar to #13833, allow a user to confirm an export in case there are more than 25 dataobjects exported (to avoid unnecessary processing if the user did not pay attention to the number of exported dataobjects, typically clicking "Export CSV" without realizing it can export thousands of occurrences)

image

### CODE REVIEW

File: bundles/AdminBundle/public/js/pimcore/element/helpers/gridColumnConfig.js [hunk 1]
```diff
@@ -756,7 +756,19 @@ pimcore.element.helpers.gridColumnConfig = {
                 var rdata = Ext.decode(response.responseText);
 
                 if (rdata.success && rdata.jobs) {
-                    this.exportProcess(rdata.jobs, rdata.fileHandle, fieldKeys, true, settings, exportType);
+                    var exportSize = rdata.jobs.reduce((a, b) => a + b.length, 0)
```
Comment (robertSt7) [inline bundles/AdminBundle/public/js/pimcore/element/helpers/gridColumnConfig.js]:
```suggestion
                    const exportSize = rdata.jobs.reduce((a, b) => a + b.length, 0)
```

**Comment by SamyMP** [comment]
@robertSt7 I wasn't sure if I needed to use var to keep consistency with existing codebase, or const to meet new JavaScript standards, thank you for the suggestion

**Comment by robertSt7** [comment]
> @robertSt7 I wasn't sure if I needed to use var to keep consistency with existing codebase, or const to meet new JavaScript standards, thank you for the suggestion

@SamyMP No problem

**Comment by robertSt7** [comment]
@SamyMP Thanks for the PR

**Pull request merged**